website/content/sdk/installation.md

```diff
@@ -4,37 +4,39 @@ sidebar_position: 1
 ---
-The OpenBB SDK provides programmatic access to all Terminal functions and much more. This layer of code allows users to build their own tools and applications on top of the existing architecture. Follow these steps to install on a local machine.
+The OpenBB SDK provides programmatic access to all Terminal functions and much more. This layer of code allows users to build their own tools and applications on top of the existing architecture.
 
 :::note
 If the OpenBB Terminal has already been installed from source code, no additional installation steps are required. **You already have the SDK installed.**
+Verify the installation by following [these steps](#verify-installation).
 :::
 
-## Minimal Installation
-
-For users who have basic developer tools installed and are using Python through a virtual environment, we recommend the following minimal installation method:
+## Requirements
 
-```shell
-pip install openbb
-```
-
-This method provides access to the data aggregation and charting functions of the OpenBB SDK. It does not provide access to the advanced features that are provided by the Portfolio Optimization and Machine Learning toolkits.
+The OpenBB SDK is designed to be used in a virtual environment. __You should not try installing the SDK into your system python or your base conda enviromnent__. If you are not sure what this means, please follow the instructions to install OpenBB Terminal from source [(link)](/terminal/installation/source).
 
-If need to use the SDK with the Machine Learning and Portfolio Optimization toolkits we recommend following the [Full Installation instructions](#full-installation).
+OpenBB SDK is available on [PyPI](https://pypi.org/project/openbb/). To install the SDK from PYPI you will need to meet the following requirements:
 
-## Full Installation
+- Use Python 3.8, 3.9, or 3.10
+- Use Python from a virtual environment
+- Have a C compiler installed (e.g. `gcc`, `clang`, etc.). These usually come with the operating system.
+- Have `cmake` installed on your machine. This is usually available through your package manager (e.g. `apt`, `brew`, `yum`, etc.).
 
-To access the full functionality of the SDK with all the default toolkits, we recommend installing the SDK from source code [(link)](/terminal/installation/source). This will enable you to use the SDK with all the toolkits and contribute to the OpenBB project.
+:::info
+If you encounter any issues with the installation, please try following the instructions to install OpenBB Terminal from source [(link)](/terminal/installation/source).
+:::
 
-However, if you already have [the requirements listed at the beginning of the source code installation instructions](/terminal/installation/source) installed on your system, you can alternatively install the SDK toolkits from PyPI.
+## Minimal Installation
 
-Install both ML and Portfolio Optimization toolkits with:
+To install from PyPI, activate your virtual environment and enter:
```

Comment:
The step how to activate an environment is missing here, again assuming people know what to do.

Reply:
We don't know what virtual environment technology the person is using. It can be venv pipenv conda etc. We advise following the terminal source docs - they explain how to activate a conda environment